locales/en.json

```diff
@@ -255,5 +255,8 @@
   "Total Deducted (437)": "Total Deducted (437)",
   "Total Credits (482)": "Total Credits (482)",
   "Total Refund (484)": "Total Refund (484)",
-  "I confirm that, as far as I know, all the information in this tax return is accurate": "I confirm that, as far as I know, all the information in this tax return is accurate"
-}
+  "I confirm that, as far as I know, all the information in this tax return is accurate": "I confirm that, as far as I know, all the information in this tax return is accurate",
+  "undefined Total Income": "undefined Total Income",
```

Comment:
This key means that it gets "undefined" at some point. is that happening during a test?

Reply:
so when we change/update the data, this'll happen, unless you go back to the start and enter an access code, to force it to pull in fresh user info